src/test_helpers/components.rs

```diff
@@ -34,6 +34,8 @@ pub struct WorkloadManager {
     namespaces: netns::NamespaceManager,
     /// workloads that we have constructed
     workloads: Vec<LocalWorkload>,
+    /// services that we have constructed. VIP -> SVC
+    services: HashMap<IpAddr, Service>,
```

Comment:
this is only test code which uses the default network. i specifically didn't update this here in an attempt to keep this PR smaller since it's already quite sizeable. open to adding more multi network stuff in follow up PRs?

I really just wanted the API correct and compiling here, so future PRs don't make single-network assumptions unconsciously. it was not my intent to get 100% multi network support here